docusaurus/docs/operate/configs/supplier_staking_config.md

```diff
@@ -65,6 +65,13 @@ a new `service`, then `stake_amount: 1001upokt` should be specified in the
 configuration file. This will increase the stake by `1upokt` and deduct `1upokt`
 from the `Supplier`'s account balance.
 
+The upstaking requirement is to ensure that a `Supplier` incurs a cost for
+changing the services they provide too frequently, which could lead to a poor user
+experience for `Gateways` and `Applications`.
+
+The upstake amount may be adjusted in the future based on network conditions
+and feedback from the community.
+
```

Comment:
```suggestion

```

Reply:
Would love to know why exactly 1upokt then.